common/dds-model.cpp

```diff
@@ -0,0 +1,349 @@
+// License: Apache 2.0. See LICENSE file in root directory.
+// Copyright(c) 2024 Intel Corporation. All Rights Reserved.
+
+#include "dds-model.h"
+#include "device-model.h"
+#include "ux-window.h"
+#include <rsutils/json.h>
+#include <rsutils/json-config.h>
+#include <rsutils/os/special-folder.h>
+
+#include <iostream>
+#include <fstream>
+
+using namespace rs2;
+using rsutils::json;
+using rsutils::type::ip_address;
+
+dds_model::dds_model( rs2::device dev )
+    : _device( dev )
+    , _window_open( false )
+    , _no_reset( false )
+    , _set_defult( false )
+    , _dds_supported( false )
+{
+    if( check_DDS_support() )
+    {
+        _defult_config = get_eth_config( _device, DEFULT_VALUES );
+        _current_config = get_eth_config( _device, ACTUAL_VALUES );
```

Comment:
We don't need the first parameter, it is always `_device` and can be used internally.